tests/runtest.py

```diff
@@ -21,7 +21,7 @@ class TestBase:
     TEST_SUCCESS_FIXED = -8
 
     objdir = 'objdir' in os.environ and os.environ['objdir'] or '..'
-    uftrace_cmd = objdir + '/uftrace --no-pager --no-event -L' + objdir
+    uftrace_cmd = objdir + '/uftrace --no-pager --no-event --color=no -L' + objdir
```

Comment:
Hmm..? I don't think it's needed since we run test with `popen()` redirecting stdout/stderr to a pipe.  The init code should detect it not being a terminal and disable color.